site/docs/hive.md

```diff
@@ -72,6 +72,15 @@ catalog.createTable(tableId, schema, spec, tableProperties);
 
 The table level configuration overrides the global Hadoop configuration.
 
+#### Hive on Tez configuration
+
+To use the Tez engine on Hive(requires version >= `3.1.3`), Tez needs to be upgraded to >= `0.10.1` which contains a necessary fix [Tez-4248](https://issues.apache.org/jira/browse/TEZ-4248).
```

Comment:
There is no `3.1.3` release ATM. Did you mean 3.1.2?

Reply:
Yes. I just changed to 3.1.2